torchio/data/sampler/uniform.py

```diff
@@ -24,7 +24,7 @@ def _generate_patches(
         patches_left = num_patches if num_patches is not None else True
         while patches_left:
             index_ini = [
-                torch.randint(x + 1, (1,)).item()
+                torch.randint(int(x) + 1, (1,)).item()
```

Comment:
Why was this necessary?

Reply:
I guess this has never been a problem, so I removed it. But in my laptop (Python 3.8.9, Pytorch 1.4.0) this raises an error: `torch.randint(np.int64(5) + 1, (1,))` so I had to cast it to int. In my other computer (newer version of Pytorch) it doesn't raise an error.

Reply:
Why not just leave it as it was? If it's an issue, it should be addressed separately as it is out of the scope of this PR.